src/main/java/org/jenkinsci/plugins/workflow/support/actions/LogActionImpl.java

```
@@ -110,8 +101,8 @@
     private String charset;
 
     private LogActionImpl(FlowNode parent, Charset charset) {
-        if (!isRunning(parent)) {
-            throw new IllegalStateException("cannot start writing logs to a finished node " + parent);
+        if (!parent.isActive()) {
+            LOGGER.log(Level.WARNING, null, new IllegalStateException("cannot start writing logs to a finished node " + parent + " " + parent.getDisplayFunctionName() + " in " + parent.getExecution()));
```

Safe to swallow the exception?